Bottom: ff93c0dff87a02a29791cdb51e6c1fb27b48ffea
Top:    f089a8416c9dd82c0d138e31743ef84f25c4bd11
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-07 16:13:11 +0200

Refresh of toy-with-a-gui

---

diff --git a/cmake/ExternalProjects.cmake b/cmake/ExternalProjects.cmake
index 9520b46..97738b3 100644
--- a/cmake/ExternalProjects.cmake
+++ b/cmake/ExternalProjects.cmake
@@ -1,16 +1,5 @@
 include(FetchContent)
 
-#FetchContent_Declare(
-#  imgui
-#  GIT_REPOSITORY https://github.com/ocornut/imgui
-#  GIT_TAG v1.83
-#)
-#FetchContent_GetProperties(imgui)
-#if(NOT imgui_POPULATED)
-#  FetchContent_Populate(imgui)
-##  add_subdirectory(${imgui_SOURCE_DIR} ${imgui_BINARY_DIR})
-#endif()
-
 FetchContent_Declare(
   fltk
   GIT_REPOSITORY https://github.com/fltk/fltk
@@ -23,8 +12,6 @@ if(NOT fltk_POPULATED)
   add_subdirectory(${fltk_SOURCE_DIR} ${fltk_BINARY_DIR})
 endif()
 
-
-
 FetchContent_Declare(
   googletest
   GIT_REPOSITORY https://github.com/google/googletest.git
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index e787794..d46203b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,4 +1,5 @@
 add_subdirectory(modules)
+
 add_executable(rpn
 	rpn.cpp
 )
@@ -6,12 +7,14 @@ target_compile_options(rpn
 	PRIVATE
 		${module_compile_options}
 )
-target_link_libraries(rpn
-	calculator
-	fltk
-)
+# These aren't properly set by fltk itself.
 target_include_directories(rpn
 	PRIVATE
-		${fltk_SOURCE_DIR}
-		${fltk_BINARY_DIR}
+		${fltk_SOURCE_DIR} # Default include directory
+		${fltk_BINARY_DIR} # Contains the build information header
+)
+target_link_libraries(rpn
+	PRIVATE
+		calculator
+		fltk
 )
